pkg/auth/authorizer/abac/abac.go

```diff
@@ -52,7 +53,46 @@ func (p policyLoadError) Error() string {
 type policyList []*api.Policy
 
 // TODO: Have policies be created via an API call and stored in REST storage.
-func NewFromFile(path string) (policyList, error) {
+func NewFromFile(path string) (*policyList, error) {
```

Comment:
If you do polling, make the flag be something like `--abac-file-poll-period-seconds` or something like that.